docs/guides/js-waku/lightpush-filter.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Send and Receive Messages Using Light Push and Filter
+---
+
+This guide provides detailed steps to create a light node, send messages using the [Light Push protocol](/overview/concepts/protocols#light-push), and receive messages using the [Filter protocol](/overview/concepts/protocols#filter).
+
+## Create a Waku Node
+
+Set up a Waku node by creating a light node, connecting to network peers with `Light Push` and `Filter` enabled, choosing a [content topic](/overview/concepts/content-topics), and creating an `encoder` and `decoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+
+```js
+import {
+    createLightNode,
+    waitForRemotePeer,
+    Protocols,
+    createEncoder,
+    createDecoder
+} from "@waku/sdk";
+
+// Create and start a light node
+const node = await createLightNode({ defaultBootstrap: true });
```

Comment:
Should we mention that `defaultBootstrap` means that PROD fleet will be used with already set-up nodes?

Reply:
<img width="860" alt="Screenshot 2023-07-10 at 10 26 18" src="https://github.com/waku-org/docs.waku.org/assets/47832826/a5b931b1-f167-4a92-ac6c-4f30a2417395">

I already mentioned something similar in the quickstart guide. Should I do it again in the lightpush and relay guides?

Reply:
Maybe we can solve it this way - https://github.com/waku-org/docs.waku.org/pull/82/files#r1257976132

Reply:
Just saw your other comment, let's resolve this and conclude there